README.md

```diff
@@ -1,7 +1,43 @@
+# App Configuration Sync action
+
+This action syncs configuration files in the repository to an App Configuration instance. This enables scenarios where the App Configuration instance is automatically updated when changes are made through GitHub workflows.
+
+JSON, YAML, and .properties files are supported. For the full list of action inputs, see [Inputs](./action.yml).
+
+## Connection string
+
+The connection string for the App Configuration instance should be stored as a [secret](https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables) in the GitHub repository.  The secret should be used in the workflow.
+
+## Usage example
+
+The following example updates the App Configuration instance each time a change is made to `appsettings.json` in the `master` branch.
+
+```yaml

```

Comment:
Can we put a comment at the top of this example

```
# File: .github/workflows/workflow.yml
```

I saw it in other Azure examples.